code/modules/clothing/shoes/magboots.dm

```diff
@@ -61,7 +65,7 @@
 		REMOVE_TRAIT(user, TRAIT_MAGPULSE, "magboots")
 
 /obj/item/clothing/shoes/magboots/negates_gravity()
-	return flags & NOSLIP
+	return magpulse
```

Comment:
This also should still be a noslips check most likely

Reply:
For magboots, magpulse and no_slip are always going to be the same value, so there'd be no difference

Reply:
Agreed, it should be consistent, even if they're supposed to always be the same var